conda_pypi/cli/install.py

```diff
@@ -50,7 +51,14 @@ def configure_parser(parser: _SubParsersAction) -> None:
     install.add_argument(
         "--override-channels",
         action="store_true",
-        help="Do not search default or .condarc channels. Will search pypi.",
+        help="Do not search default or .condarc channels. Will search PyPI.",
+    )
+    install.add_argument(
+        "-i",
+        "--index-url",
+        dest="index_urls",
+        action="append",
+        help="Add a PyPI index URL (can be used multiple times).",
```

Comment:
@danyeaw, just wanted to make sure you are alright with us _not_ adding `--index`, but `-i / --index-url`.

Reply:
@jaimergp yup, I'm good with it.